config/routes.rb

```diff
@@ -43,6 +43,18 @@
     collection do
       delete 'destroy_multiple'
     end
+
+    member do
+      post 'reserve'
+    end
+  end
+
+  # TODO
+  resources :reservations, only: [:destroy] do
```

Comment:
Yeah that'll work - destroy is just a DELETE request against `reservations/:id` though